testing/trino-benchto-benchmarks/README.md

```diff
@@ -84,10 +90,20 @@ tpch_medium: tpcds_10gb_txt
 
 With the scene set up as in the previous section, the benchmark can be run with:
 ```bash
-./mvnw clean package -pl :trino-benchto-benchmarks
-java -Xmx1g -jar trino-benchto-benchmarks/target/trino-benchto-benchmarks-*-executable.jar \
-    --sql trino-benchto-benchmarks/src/main/resources/sql \
-    --benchmarks trino-benchto-benchmarks/src/main/resources/benchmarks \
-    --activeBenchmarks=presto/tpch --profile=presto-devenv \
-    --overrides overrides.yaml
+# set JAVA_HOME to the JDK 8 home dir
+local_repo=$(./mvnw -B help:evaluate -Dexpression=settings.localRepository -q -DforceStdout)
```

Comment:
Can we assume it's always `$HOME/.m2/repository`? That'll make the example simpler.